test/ServerComparison.FunctionalTests/Helpers.cs

```diff
@@ -12,5 +12,20 @@ public static string GetApplicationPath()
         {
             return Path.GetFullPath(Path.Combine("..", "..", "..", "..", "..", "ServerComparison.TestSites"));
         }
+
+        public static string GetConfigContent(ServerType serverType, string iisConfig, string nginxConfig)
```

Comment:
IIS has multiple config files depending on the scenario. E.g. HelloWorld vs NTLM